src/main/resources/application.properties

```diff
@@ -1,13 +0,0 @@
-server.port=9092
-
-spring.datasource.url=jdbc:h2:file:./data/demo
-spring.datasource.driverClassName=org.h2.Driver
-spring.datasource.username=admin
-spring.datasource.password=admin
-spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
-spring.jpa.hibernate.ddl-auto=update
-spring.jpa.show-sql=true
-spring.h2.console.enabled=true
-spring.h2.console.path=/h2-console
-spring.h2.console.settings.trace=false
-spring.h2.console.settings.web-allow-others=false
```

Comment:
please remove entire file and directory.
its like empty file in `src` directory

Reply:
🔴 **Remove hole directory if it's useless** 

![removeplease](https://user-images.githubusercontent.com/58255161/217086553-fdc9fe6a-1bc7-4fc0-92a6-4162214f3651.png)